projects/rv-agent/repos

```diff
@@ -0,0 +1,5 @@
+# List of repos to setup.
+# Format:
+# <github repo name> <pull on each `brew barkly`?>
+
+RV-agent
```

Comment:
Not sure if case matters, but should be `RV-Agent` if so

Reply:
github is not case sensitive

Reply:
but good eye @dominiclaudate  ;)